CMakeLists.txt

```diff
@@ -12,12 +12,11 @@
 # You should have received a copy of the GNU General Public License along with
 # SymCC. If not, see <https://www.gnu.org/licenses/>.
 
-cmake_minimum_required(VERSION 3.5)
-project(SymbolicCompiler)
+cmake_minimum_required(VERSION 3.16)
+project(Symcc)
```

Comment:
Wow, `SymbolicCompiler` is a really old name! Git says it's from the very first commit :sunglasses: 

Let's try to use the usual spelling `SymCC`, shall we? Or does CMake complain about it for some reason?